testing/reindev/2.8.1_06-Fabric/build.gradle

```diff
@@ -68,5 +112,4 @@ repositories {
 
 java {
     sourceCompatibility = targetCompatibility = JavaVersion.VERSION_1_8
-    toolchain.languageVersion = JavaLanguageVersion.of(8)
```

Comment:
Gradle randomly started complaining about this, saying `languageVersion` is final?

Reply:
-_-